### PR TITLE
Docs

### PR DESCRIPTION
**Pull request by abhiskk**
## Motivation and Context

Adds documentation for habitat-api, to view the docs follow instructions in `docs/README.md` and view the html files from the build folder.

image

## Types of changes

- Updation of docstrings.

## Checklist

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have completed my CLA (see **CONTRIBUTING**)
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -155,5 +154,3 @@ The Habitat project would not have been possible without the support and contrib
 ## License
 Habitat-API is MIT licensed. See the LICENSE file for details.
 
-## References
```
Comment (dhruvbatra) [inline README.md]: Why are we taking the reference out?

**Comment by abhiskk** [comment]
We report the citation under the [citing habitat](https://github.com/facebookresearch/habitat-api#citing-habitat) section. The references section looks redundant with this, I tried searching for any locations the references section is being used and could not find one. I think the references section was put in circa the arxiv upload going live.

**Comment by abhiskk** [comment]
On second thought having the references section makes it easier for someone looking for the paper, bringing it back.

**Comment by mathfac** [comment]
Looks terrific, thank you for doing this! Left some comments and worth to rebase on master and maybe include check or doc regeneration step to CI.

File: habitat/sims/habitat_simulator.py [hunk 2]
```diff
@@ -138,8 +138,8 @@ def get_observation(self, sim_obs):
 
 
 @registry.register_simulator(name="Sim-v0")
-class HabitatSim(Simulator):
-    """Simulator wrapper over habitat-sim
+class HabitatSim(habitat.Simulator):
```
Comment (erikwijmans) [inline habitat/sims/habitat_simulator.py]:
```suggestion
class HabitatSim(Simulator):
```
This is why tests are failing

Comment (abhiskk) [reply habitat/sims/habitat_simulator.py]: thanks, this snuck in when I was resolving merge conflicts.

File: docs/source/conf.py [hunk 3]
```diff
@@ -0,0 +1,208 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# pythia documentation build configuration file, created by
```
Comment (mathfac) [inline docs/source/conf.py]:
```suggestion
# Habitat documentation build configuration file, created by
```

File: README.md [hunk 4]
```diff
@@ -1,5 +1,5 @@
 <p align="center">
-  <img width = "50%" src='docs/img/habitat_logo_with_text_horizontal_blue.png' />
+  <img width = "50%" src='res/img/habitat_logo_with_text_horizontal_blue.png' />
```
Comment (mathfac) [inline README.md]: Do we need rename or move `docs/img` to `res/img`?

File: docs/source/tutorials/quickstart.md [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+
+For installing Habitat-Sim and Habitat-API follow instructions [here](https://github.com/facebookresearch/habitat-api#installation).
+
+## Example
+
+In this example we will setup a PointNav task in which the agent is tasked to go from a source location to a target location. For this example the agent will be you (the user). You will be able to step around in an environment using keys.
+
+For running this example both Habitat-Sim and Habitat-API should be installed successfully. The data for scene should also be downloaded (steps to do this are provided in the installation instructions of Habitat-API). Running the code below also requires installation of cv2 which you can install using: `pip install opencv-python`.
```
Comment (mathfac) [inline docs/source/tutorials/quickstart.md]: Can we link to particular point in README how to download data?

File: README.md (hunk 4 above)
Comment (abhiskk) [reply README.md]: I had moved the imgs folder from `docs/` to a `res/` folder under the main repository, hence this renaming is needed. Does this answer your question?

Comment (mathfac) [reply README.md]: ow, finally found that in changed file list.

**Pull request merged**